pulp_container/app/tasks/sync_stages.py

```diff
@@ -674,3 +675,8 @@ def _post_save(self, batch):
             BlobManifest.objects.bulk_create(blob_manifests, ignore_conflicts=True)
         if manifest_list_manifests:
             ManifestListManifest.objects.bulk_create(manifest_list_manifests, ignore_conflicts=True)
+
+        # the nature of manifest lists can be determined after creating the listed manifest relation
+        for ml in manifest_lists:
```

Comment:
is operations is not really related to relation creation, can you rename the variable from manifest_lits to something more explicit, like manifests_lists_nature_to_init

Reply:
I was trying to say in the comment that we can run this initialization only after creating the relations between listed manifests and manifest lists. It appears that the combination of the comment and variable name causes confusion. 😞 Let me remove the comment and rename the variable.

Reply:
i think the comment is helpful. you can leave it + rename the var ;)

Reply:
Okay, I decided to rewrite the comment instead:
```
        # after creating the relation between listed manifests and manifest lists,
        # it is possible to initialize the nature of the corresponding manifest lists
```
How do you like it now? The suggested explicit name is hard to read and understand. We iterate over manifest lists and thus it does make sense to call the list "manifest_lists".